pkg/shell/base_index.js

```diff
@@ -409,8 +408,8 @@ function Router(index) {
                     source = register(child);
                 }
                 if (source) {
-                    const reply = $.extend({ }, cockpit.transport.options,
-                                           { command: "init", host: source.default_host, "channel-seed": source.channel_seed }
+                    const reply = cockpit.extend({ }, cockpit.transport.options,
+                                                 { command: "init", host: source.default_host, "channel-seed": source.channel_seed }
```

Comment:
OK, but `Object.assign()` should do the same, and of course the l33t ES6 way is to use the spread operator :grin:

Reply:
yes, I am aware of all three options, but went with this one for no good reason. But I think we might want to replace all `cockpit.extend` with something directly from JS. (in followup)

Reply:
Thanks! I added it to my "later" todo list, let's not block this PR on such neglibilities.